cannon/mipsevm/tests/evm_common_test.go

```diff
@@ -515,8 +516,16 @@ func TestEVMFault(t *testing.T) {
 				// set the return address ($ra) to jump into when test completes
 				state.GetRegistersRef()[31] = testutil.EndAddr
 
+				// TODO: Hardcode here to achieve the expected result.
+				// Future improvements might involve having the Go VM indicate attempted memory accesses
+				// and caching this info to generate proofs for the solidity VM.
+				memoryProofAddress := uint32(0)
```

Comment:
Looks like we don't need any proof address for the other test cases.  If you configure the `memoryProofAddresses` in the `cases` array we can drop this for clarity.